common-pg/src/main/scala/com/socrata/pg/store/RollupManager.scala

```diff
@@ -262,21 +288,16 @@ class RollupManager(pgu: PGSecondaryUniverse[SoQLType, SoQLValue], copyInfo: Cop
       "dataset_id" -> copyInfo.datasetInfo.systemId.underlying,
       "rollupName" -> rollupInfo.name.underlying) {
       val soqlAnalysis = analysesToSoQLType(rollupAnalyses)
-      val sqlCtx = Map[SqlizerContext, Any](
-        SqlizerContext.CaseSensitivity -> true,
-        SqlizerContext.LeaveGeomAsIs -> true
-      )
-
-      val dsRepMap: Map[QualifiedUserColumnId, SqlColumnRep[SoQLType, SoQLValue]] =
-        dsSchema.values.map(ci => QualifiedUserColumnId(None, ci.userColumnId) -> SoQLIndexableRep.sqlRep(ci)).toMap
-
-      val tableMap = Map(TableName.PrimaryTable -> copyInfo.dataTableName) // TODO: FIX ME
-      val selectParamSql = Sqlizer.sql(Tuple3(soqlAnalysis, tableMap, rollupReps))(
-        rep = dsRepMap,
-        Map.empty,
-        setParams = Seq(),
-        ctx = sqlCtx,
-        stringLit => SqlUtils.escapeString(pgu.conn, stringLit))
+
+     val selectParamSql = QueryServerHelper.sqlize(// scalastyle:ignore method.length parameter.number cyclomatic.complexity
+       pgu,
+       copyInfo.datasetInfo,
+       soqlAnalysis,
+       None, //reqCopy: Option[String],
+       None, //rollupName: Option[RollupName],
+       false, // obfuscateId: Boolean,
```

Comment:
It seems like this is a brand new function that's used only once, curious why you have these unused params in them from the start

Reply:
I added comment to explain this.  TODO: Refactor QueryServer.runQuery to use this function